_meeting-notes/2019-12-03.md

```diff
@@ -0,0 +1,166 @@
+Test Ground Weekly (45 mins)
+===
+
+###### tags: `IPFS` `Testing`
+
+:::info
+- **Host:** @daviddias
+- **Location:** https://protocol.zoom.us/j/181391682
+- **Date:** Dec 3, 2019 6:00 PM (UTC)
+- **Participants:**
+    - @daviddias
+    - @nonsense
+    - @raulk
+    - @jimpick
+    - @momack2
+    - @stebalien
+- **Agenda**
+  ~~1. Sprint Update `00min`~~
+  2. Identify blockers / Answer Questions `30min`
+  3. This week's planning `15min`
+- **Reference:** 
+  - [Last week meeting minute](https://github.com/ipfs/testground/pull/69)
+  - [KanBan Board](https://app.zenhub.com/workspaces/test-ground-5db6a5bf7ca61c00014e2961/board)
+  - [Testing Infra Design](https://github.com/ipfs/testground/blob/master/docs/SPEC.md)
+  - [Testing Inventory](https://github.com/ipfs/testground/blob/master/docs/test-inventory.md)
+  - [Testing Improvement Tracker](https://docs.google.com/spreadsheets/d/1xyqyGUF-oe3x9ln88YonVeOMWWdknik74lVgL_3dBY8/edit#gid=0)
+
+:::
+
+## :mega: Update (Done, Shipped, Made Progress)
+> To be reviewed Async
+
+- ==@raulk==
+    - Little progress since I was on holiday last week, and now combating my backlog.
+    - Thought about multiversion test runs; planning to write a design blueprint.
+        - Gist: new concept "ensemble". An ensemble is a cohort of runs that run with the same ID but using different build outputs.
+        - Depends on tracking the dependency graph. Who can own this?
+    - Refactored the DHT plan. Currently a WIP PR; need to push it over the finish line this week.
+    - Arranging/negotiating travels for testground hack week.
+- ==@daviddias==
+    - Created Onboarding Work Plan for Henrique
+    - Coordinated the handing of the Infra Responsibilities from Jim to Anton
+    - Created the MVP/Primary Goal issue https://github.com/ipfs/testground/issues/196
+    - Made the plans and handled the logistics for the next Team "Onboarding" Hack Week
+    - Created a PRD for the TestGround Dashboard
+    - Reviewed PRs
+    - Did not have any tasty Thanksgiving Holidays food (we need that holiday here! :D)
+- ==@stebalien==
+    - Out Nov 27(Wed) - Nov 29(Fri) for Thanksgiving
+- ==@jimpick==
```

Comment:
@jimpick, did you had a chance to do:

- Do the Makefile for Steven's Sidecar built

?? Will you carry it for this week? 

Ref: https://github.com/ipfs/testground/blob/master/_meeting-notes/2019-11-26.md#assignments

Reply:
On Friday, I mentioned sidecar and you asked me not to work on it so I removed myself from the issue... https://github.com/ipfs/testground/issues/176

I can still do it - the Makefile seems simple enough, but I'm sure there are puzzles to figure out when it comes time to make sure that the right version is deployed before we run the runner (local and docker swarm).

Reply:
My apologies, I might have misunderstood. My intention was **avoid trying to duplicate or create overlapping work with what @Stebalien is doing/owning** and not advising to **drop a commitment made to a colleague without notice**.

I believe that @Stebalien is still counting on you to make that Makefile. It is best to check with him and update this mini thread and your weekly goals respectively.